docs/conf/scalar-prometheus-custom-values.yaml

```
@@ -34,6 +39,25 @@ grafana:
       # -- https://grafana.com/docs/grafana/latest/administration/configuration/#allow_embedding
       allow_embedding: false
 
+  # -- Additional data source configurations
+  additionalDataSources:
+  - name: Prometheus
+    type: prometheus
+    uid: prometheus
+    url: http://scalar-monitoring-kube-pro-prometheus.monitoring.svc.cluster.local:9090/
```

`.monitoring.svc.cluster.local` should be omitted. If a Prometheus server is deployed to a namespace that is not `monitoring`, the URL will not work. Also, `.cluster.local` is just the default, but it is not always used (although it is almost always the default).

```suggestion
    url: http://scalar-monitoring-kube-pro-prometheus:9090/
```

---

Thank you for your suggestion! I applied it.